src/wp-admin/about.php

```diff
@@ -609,3 +609,9 @@
 
 /* translators: 1: WordPress version number, 2: Link to update WordPress */
 __( 'Important! Your version of WordPress (%1$s) will stop receiving security updates in the near future. To keep your site secure, please <a href="%2$s">update to the latest version of WordPress</a>.' );
+
+/* translators: The major version of WordPress for this branch. */
+__( 'This is the final release of WordPress %s' );
```

Comment:
Should the translator comment include the placeholder for consistency with other strings?
```
/* translators: %s: The major version of WordPress for this branch. */
```

Reply:
Thanks Sergey, that's a good idea.

All the strings above with multiple placeholders include `1:` and `2:` rather than the full placeholder. Should they be updated in another ticket: probably the docs improvement ticket?

Reply:
> All the strings above with multiple placeholders include `1:` and `2:` rather than the full placeholder. Should they be updated in another ticket: probably the docs improvement ticket?

Good question, historically translator comments use non-numbered placeholders directly (e.g. `%s:` or `%d:`), but just `1:` or `2:` for numbered placeholders like `%1$s` or `%2$d`. [#9112 Use gettext's contexts and extracted comments instead of our pipe syntax](https://core.trac.wordpress.org/ticket/9112) appears to be the earliest ticket introducing translator comments using this format.

I don't know if that's something we should fix, my guess is that `1:` is just a bit more readable as a hint than `%1$s:`, though the latter should be OK too. I think this could use some input from the Polyglots team.